BoardingPass/HorizontalSlideAnimatedTransition.swift

```diff
@@ -79,20 +60,18 @@ extension HorizontalSlideAnimatedTransiton: UIViewControllerAnimatedTransitionin
             presented.view.transform = CGAffineTransform.identity
         }
         let completion = { (completed: Bool) in
-            if completed {
-                presented.view.transform = CGAffineTransform.identity
-            }
-            else {
-                presenting.view.transform = CGAffineTransform.identity
-            }
+            presented.view.transform = CGAffineTransform.identity
+            presenting.view.transform = CGAffineTransform.identity
             transitionContext.completeTransition(completed)
         }
-        UIView.animate(withDuration: transitionDuration(using: transitionContext), delay: 0.0,
-                                   options: [.beginFromCurrentState],
-                                   animations: animations, completion: completion)
+        UIView.animate(
```

Comment:
If the library is iOS 9 plus a property animator is going to get better results for interactive animations.